modules/python/python-logmsg.c

```diff
@@ -75,8 +75,9 @@ _py_log_message_subscript(PyObject *o, PyObject *key)
 
   PyLogMessage *py_msg = (PyLogMessage *) o;
   PyObject *value = _get_value(py_msg, name, py_msg->cast_to_bytes);
-  if (!value)
+  if (value == Py_None)
     {
+      Py_XDECREF(value);
```

Comment:
_get_value() can return NULL as well

Reply:
I've refactored get_value() not to return NULL unless it sets an exception.